### PR TITLE
Added Useful Xcode Scripts

### PR DESCRIPTION
**Pull request by aashishtamsya**
## Checklist

- [x] Only one project is in this pull request
- [x] Addition in chronological order (bottom of category)
- [ ] `Travis CI` pass
- [ ] Supports iOS 7 or later
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by dkhamsing** [comment]
Link is missing

**Comment by lfarah** [comment]
Also please add it under [Other Xcode section](https://github.com/vsouza/awesome-ios#other-xcode)

File: README.md [hunk 1]
```diff
@@ -131,6 +131,7 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
 - [Xcode](#xcode)
     - [Plugins](#plugins)
     - [Themes](#themes)
+	- [Xcode Scripts](#xcode-scripts)
```
Comment (dkhamsing) [inline README.md]: please undo this change

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">Please only add one library per Pull Request</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by aashishtamsya** [comment]
Added only add one library per Pull Request

**Comment by aashishtamsya** [comment]
Added only add one library per Pull Request

**Comment by dkhamsing** [comment]
Thx
